app/views/fandoms/index.html.erb

```diff
@@ -1,3 +1,6 @@
+<% collection_key = @collection.nil? ? "NULL" : @collection.id %>
+<% media_key = @medium.nil? ? "NULL" : @medium.name %>
+<% cache("/v1/media_pages/#{collection_key}/#{media_key}", :expires_in => 1.hour) do %>
```

Comment:
Can you use the new syntax here? Also, when you do fragment caching like this, please update the indenting of the file to make it clearer what's inside the caching -- basically, everything between the start and end should move over 1 tab (2 spaces).